package.json

```diff
@@ -18,6 +18,7 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "@typescript-eslint/parser": "1.10.2",
```

Comment:
Simple question, shall we use a range here?

Reply:
ah I didn't notice that it locked it when I added it. thanks for that

Reply:
is `^1.10.2` good enough or were you thinking something wider?

Reply:
Yeah, caret ranges are the most common ranges.